Error has occurred !!
```
parse/parser.go:381:16: too few values in parseError literal
parse/parser.go:389:16: too few values in parseError literal
```
plz fix the error
you can check this in here https://travis-ci.org/DE-labtory/koa/builds/483891010?utm_source=github_status&utm_medium=notification

--- a/parse/parser.go
+++ b/parse/parser.go
@@ -379,6 +379,8 @@ func parsePrefixExpression(buf TokenBuffer) (ast.Expression, error) {
 			return nil, parseError{
 				tok.Type,
 				fmt.Sprintf("parsePrefixExpression() - Invalid prefix of %s", right.String()),
+				tok.Line,
+				tok.Column,
 			}
 		}
 	case ast.Minus:
@@ -387,6 +389,8 @@ func parsePrefixExpression(buf TokenBuffer) (ast.Expression, error) {
 			return nil, parseError{
 				tok.Type,
 				fmt.Sprintf("parsePrefixExpression() - Invalid prefix of %s", right.String()),
+				tok.Line,
+				tok.Column,
 			}
 		}
 	}

--- a/parse/parser_internal_test.go
+++ b/parse/parser_internal_test.go
@@ -1008,7 +1008,7 @@ func TestMakePrefixExpression(t *testing.T) {
 				},
 				0,
 			},
-			expectedErr: errors.New("MINUS, parsePrefixExpression() - Invalid prefix of true"),
+			expectedErr: errors.New("[line 0, column 0] parsePrefixExpression() - Invalid prefix of true"),
 		},
 		{
 			buf: &mockTokenBuffer{
@@ -1018,7 +1018,7 @@ func TestMakePrefixExpression(t *testing.T) {
 				},
 				0,
 			},
-			expectedErr: errors.New("BANG, parsePrefixExpression() - Invalid prefix of \"hello\""),
+			expectedErr: errors.New("[line 0, column 0] parsePrefixExpression() - Invalid prefix of \"hello\""),
 		},
 	}
 

